game/models.py

```diff
@@ -585,13 +586,44 @@ class Meta:
         )
 
 
-class CompetitionManager(models.Manager):
+class CompetitionMixin(object):
```

Comment:
IMO `ongoing` and `upcoming` are "initial" filters on the queryset. It wouldn't make any sense to use `ongoing()` and `upcoming()` in a chain call, like `Competition.objects.ongoing().upcoming()`.

It's also shorter to write `Competition.ongoing.all()` than `Competition.objects.ongoing()`. :)